lib/gateway/Shard.js

```diff
@@ -42,6 +45,10 @@ class Shard extends EventEmitter {
         this.hardReset();
     }
 
+    get latency() {
+        return this.lastHeartbeatSent && this.lastHeartbeatReceived ? this.lastHeartbeatReceived - this.lastHeartbeatSent : null;
```

Comment:
`Infinity` would certainly work, but I'm wondering if there's a better way to signify that the value the user is asking for doesn't make sense.